llvm/lib/Target/AIE/AIEBasePipelinerLoopInfo.cpp

```diff
@@ -52,9 +52,14 @@ cl::opt<bool> LoopWholeLoopGuard(
 
 cl::opt<int> PostPipelinerCandidateLimit(
     "aie-postpipeliner-limit",
-    cl::desc("II below which postpipeliner preference kicks in"), cl::init(2),
+    cl::desc("II below which postpipeliner preference kicks in"), cl::init(4),
     cl::Hidden);
 
+cl::opt<int> PostPipelinerCutoff(
+    "aie-postpipeliner-cutoff",
+    cl::desc("II below which high stage count is left to the postpipeliner"),
+    cl::init(11), cl::Hidden);
```

Comment:
Checking:
 - PostPipelinerCandidateLimit means we will leave the loop intact if a schedule with `II<=PostPipelinerCandidateLimit` was found. No questions asked
 - PostPipelinerCutoff means we will leave the loop intact if a schedule was found below that II, but only if the stage count is high. (I guess greater than `LoopMaxStageCount`?)

Reply:
There's no actual difference, but I would phrase the second one as 'when a high stagecount is found but only if the II is below cutoff.'